src/cljs/commiteth/manage_payouts.cljs

```diff
@@ -1,91 +1,356 @@
+    [:div.mb3.br3.shadow-6.bg-white.tc.pa5
+     [:h3.pg-book "Nothing to merge"]
+     [:p "Here you will see the claims waiting to be merged"]]
+    (into [:div]
+          (for [bounty bounties
+                :let [claims (:claims bounty)]]
+            ^{:key (:issue_id bounty)}
+            [:div.mb3.shadow-6
+             [:div.pa3.bg-white.br3.br--top
+              [bounty-card bounty]
+              [:div.mt3.f6 [bounty-balance bounty]]]
+             [:div.pa3.bg-sob-tint.br3.br--bottom
+              (for [claim (:claims bounty)]
+                ^{:key (:pr_id claim)}
+                [claim-card bounty claim {:render-view-claim-button? true}])]]))))
 
 (defn bounty-stats [{:keys [paid unpaid]}]
   [:div.cf
-   [:div.fl-ns.w-50-ns.tc.pv4
-    [:div.ttu.tracked "Open"]
-    [:div.f2.pa2 (common/usd-string (:combined-usd-value unpaid))]
-    [:div (:count unpaid) " bounties"]]
+   [:div.fl-ns.w-33-ns.tc.pv4
+    #_[:div.ttu.tracked "Paid"]
+    [:div.f3.pa2 (common/usd-string (:combined-usd-value paid))]
+    [:div.ph4 "Invested so far"]]
+
+   [:div.fl-ns.w-33-ns.tc.pv4
+    [:div.f3.pa2 (:count paid)]
+    [:div.ph4 "Bounties solved by contributors"]]
 
-   [:div.fl-ns.w-50-ns.tc.pv4
-    [:div.ttu.tracked "Paid"]
-    [:div.f2.pa2 (common/usd-string (:combined-usd-value paid))]
-    [:div (:count paid) " bounties"]]])
+   [:div.fl-ns.w-33-ns.tc.pv4
+    [:div.f3.pa2 (:count unpaid)]
+    [:div.ph4 "Open bounties in total"]]])
+
+(defn bounty-stats-new [{:keys [paid unpaid]}]
+  [:div.br3.bg-white.shadow-6.pa4
+   [:span.db.f3.pg-med.mb2 (common/usd-string (:combined-usd-value paid))]
+   [:span.gray "Paid for " [:span.muted-blue (:count paid) " solved bounties"]]
+   [:div.bb.b--near-white.mv3]
+   [:span.db.f3.pg-med.pt1.mb2 (common/usd-string (:combined-usd-value unpaid))]
+   [:span.gray "Open for " [:span.muted-blue (:count unpaid) " bounties"]]])
+
+(def state-mapping
+  {:opened :open
+   :funded :funded
+   :claimed :claimed
+   :multiple-claims :claimed
+   :merged :merged
+   :pending-contributor-address :pending-contributor-address
+   :pending-maintainer-confirmation :pending-maintainer-confirmation
+   :paid :paid})
+
+(defn bounty-title-link [bounty show-date?]
+  [:a {:href (common/issue-url (:repo-owner bounty) (:repo-name bounty) (:issue-number bounty))}
+   [:div.w-100.overflow-hidden
+    [:span.db.f5.pg-med.muted-blue.hover-black (:issue-title bounty)]
+    (when show-date?
+      [:span.db.mt1.f7.gray.pg-book
+       ;;{:on-click #(do (.preventDefault %) (prn (dissoc bounty :claims)))}
+       (common/human-time (:updated bounty))])]])
+
+(defn square-card
+  "A mostly generic component that renders a square with a section
+  pinned to the top and a section pinned to the bottom."
+  [top bottom]
+  [:div.aspect-ratio-l.aspect-ratio--1x1-l
+   [:div.bg-sob-tint.br3.shadow-6.pa3.aspect-ratio--object-l.flex-l.flex-column-l
+    [:div.flex-auto top]
+    [:div bottom]]])
+
+(defn unclaimed-bounty [bounty]
```

Comment:
Just one minor gripe: JS console complains abount `Warning: Every element in a seq should have a unique :key: ([commiteth.manage_payouts.unclaimed_bounty_reagentRender`